pkg/fanal/analyzer/os/release/release.go

```diff
@@ -40,9 +40,18 @@ func (a osReleaseAnalyzer) Analyze(_ context.Context, input analyzer.AnalysisInp
 		key, value := strings.TrimSpace(ss[0]), strings.TrimSpace(ss[1])
 
 		switch key {
+		case "NAME":
+			osName := strings.Trim(value, `"'`)
+			osFamily := nameToOSFamily(osName)
```

Comment:
I don’t think we need two sources for osFamily.
This can easily introduce confusion when maintaining the code and when adding support for new operating systems.

I suggest keeping id as the primary source, and using name only for normalization (post-processing).
This way it’s clear that:
	•	we always rely on id, and
	•	we fall back to name only in exceptional cases, when id doesn’t provide a reliable value.

I’ve put together a small example to illustrate my thinking:
```diff

```